test/source-sink-errors/mzcompose.py

```diff
@@ -203,15 +213,6 @@ def assert_error(self, c: Composition, error: str) -> None:
                   FROM mz_internal.mz_source_statuses
                   WHERE name = 'source1'
                 stalled true
-
-                # Sinks generally halt after receiving an error, which means that they may alternate
-                # between `stalled` and `starting`. Instead of relying on the current status, we
-                # check that there is a stalled status with the expected error.
-                > SELECT bool_or(error ~* '{error}'), bool_or(details->'namespaced'->>'kafka' ~* '{error}')
```

Comment:
OOC why is this removed? It's validating the error message contents and that the errors are namespaced. Since it's stalled, there would be some corresponding error to check for here?

Reply:
This just moved elsewhere in the same file

Reply:
Saw it's in the sink specific disruption 🤦‍♀️